uint32 drone_id